<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="LAPTOP-ASCRCERK" failures="4" tests="36" name="Sales force Automation" time="235.009" errors="0" timestamp="2025-05-30T00:38:50 CDT">
  <testcase classname="testcases.Logintest" name="loginwithValidusernameandInvalidpassword1" time="0.78"/>
  <testcase classname="testcases.Logintest" name="logwithvalidusernameandpassword2" time="2.516"/>
  <testcase classname="testcases.Logintest" name="checkRememberMe3" time="4.452"/>
  <testcase classname="testcases.Logintest" name="forgotpswd4a" time="0.771"/>
  <testcase classname="testcases.Logintest" name="forgotpswd4b" time="0.782"/>
  <testcase classname="testcases.UsermenuDropdowntest" name="selectUsermenuDropdownTC5" time="3.832"/>
  <testcase classname="testcases.UsermenuDropdowntest" name="myProfileTC6" time="6.329"/>
  <testcase classname="testcases.UsermenuDropdowntest" name="mySettingsOption7" time="8.329"/>
  <testcase classname="testcases.UsermenuDropdowntest" name="developerConsole8" time="4.589"/>
  <testcase classname="testcases.UsermenuDropdowntest" name="logoutOption9" time="4.661"/>
  <testcase classname="testcases.CreateAccounttest" name="createAccount10" time="5.911"/>
  <testcase classname="testcases.CreateAccounttest" name="createNewView11" time="4.815"/>
  <testcase classname="testcases.CreateAccounttest" name="editView12" time="6.17"/>
  <testcase classname="testcases.CreateAccounttest" name="mergeAccount13" time="6.897"/>
  <testcase classname="testcases.CreatOpportunitiestest" name="opportunitiesDropdown15" time="3.347"/>
  <testcase classname="testcases.CreatOpportunitiestest" name="createNewOpty16" time="5.498"/>
  <testcase classname="testcases.CreatOpportunitiestest" name="oppoPipeLineReport17" time="4.137"/>
  <testcase classname="testcases.CreatOpportunitiestest" name="stuckOpportunitiesReport18" time="4.871"/>
  <testcase classname="testcases.CreatOpportunitiestest" name="quaterlySummaryReport19" time="4.461"/>
  <testcase classname="testcases.Leadstest" name="leadsTab20" time="3.171"/>
  <testcase classname="testcases.Leadstest" name="leadsSelectView21" time="3.314"/>
  <testcase classname="testcases.Leadstest" name="defaultView22" time="7.461"/>
  <testcase classname="testcases.Leadstest" name="todaysLeads23" time="4.397"/>
  <testcase classname="testcases.Leadstest" name="checkNewButton24" time="5.285">
    <failure type="org.openqa.selenium.StaleElementReferenceException" message="stale element reference: stale element not found
  (Session info: chrome=137.0.7151.55)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: &amp;apos;3.141.59&amp;apos;, revision: &amp;apos;e82be7d358&amp;apos;, time: &amp;apos;2018-11-14T08:17:03&amp;apos;
System info: host: &amp;apos;LAPTOP-ASCRCERK&amp;apos;, ip: &amp;apos;192.168.1.213&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;23.0.2&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 137.0.7151.55, chrome: {chromedriverVersion: 137.0.7151.55 (254bc711794d..., userDataDir: C:\Users\mailm\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:58487}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 6a51eacda587239d989b718282fbcb15">
      <![CDATA[org.openqa.selenium.StaleElementReferenceException: stale element reference: stale element not found
  (Session info: chrome=137.0.7151.55)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'LAPTOP-ASCRCERK', ip: '192.168.1.213', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '23.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 137.0.7151.55, chrome: {chromedriverVersion: 137.0.7151.55 (254bc711794d..., userDataDir: C:\Users\mailm\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:58487}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 6a51eacda587239d989b718282fbcb15
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:285)
at org.openqa.selenium.remote.RemoteWebElement.getText(RemoteWebElement.java:166)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:51)
at jdk.proxy2/jdk.proxy2.$Proxy11.getText(Unknown Source)
at pages.Leadspage.getCheckName(Leadspage.java:123)
at testcases.Leadstest.checkNewButton24(Leadstest.java:127)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- checkNewButton24 -->
  <testcase classname="testcases.Contactstest" name="createNewContact25" time="5.912"/>
  <testcase classname="testcases.Contactstest" name="createNewViewContactPage26" time="5.455"/>
  <testcase classname="testcases.Contactstest" name="createdContactIsInTheContactpage27" time="5.867"/>
  <testcase classname="testcases.Contactstest" name="checkMyContacts28" time="3.986">
    <failure type="java.lang.AssertionError" message="Expected contact name &amp;apos;Indian&amp;apos; not found in My Contacts. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Expected contact name 'Indian' not found in My Contacts. expected [true] but found [false]
at testcases.Contactstest.checkMyContacts28(Contactstest.java:123)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- checkMyContacts28 -->
  <testcase classname="testcases.Contactstest" name="viewAContact29" time="4.352"/>
  <testcase classname="testcases.Contactstest" name="viewAContact30" time="4.613"/>
  <testcase classname="testcases.Contactstest" name="cancelButtonInCreateNewView31" time="4.757"/>
  <testcase classname="testcases.Contactstest" name="saveAndNewButton32" time="7.413"/>
  <testcase classname="testcases.RandomScenariostest" name="blockingAnEventInTheCalender36" time="6.924"/>
  <testcase classname="testcases.RandomScenariostest" name="verifyEditedLastNames34" time="6.512">
    <failure type="java.lang.AssertionError" message="did not expect [Poornitha XYZ ] but found [Poornitha XYZ ]">
      <![CDATA[java.lang.AssertionError: did not expect [Poornitha XYZ ] but found [Poornitha XYZ ]
at testcases.RandomScenariostest.verifyEditedLastNames34(RandomScenariostest.java:97)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- verifyEditedLastNames34 -->
  <testcase classname="testcases.RandomScenariostest" name="verifyTabCustomization35" time="8.036">
    <failure type="java.lang.AssertionError" message="did not expect [10] but found [10]">
      <![CDATA[java.lang.AssertionError: did not expect [10] but found [10]
at testcases.RandomScenariostest.verifyTabCustomization35(RandomScenariostest.java:129)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- verifyTabCustomization35 -->
  <testcase classname="testcases.RandomScenariostest" name="verifyFirstNameLastName33" time="5.427"/>
  <testcase classname="testcases.CreateAccounttest" name="createAccountReport14" time="0.0">
    <ignored/>
  </testcase> <!-- createAccountReport14 -->
</testsuite> <!-- Sales force Automation -->
